[TASK] Update Manage Content Page (#196)
TYPO3-Documentation/TYPO3CMS-Tutorial-Editors: Documentation/ContentElements/Content/Index.rst

..  include:: /Includes.rst.txt

..  _content-working:

================
Managing content
================

..  image:: /Images/ManualScreenshots/ContentElements/ManageContent.png

>     :alt: Manage content elements in the TYPO3 backend

Reviewer: This screenshot shows just the page module link from the left main menu and nothing else. It does not show any content element

Reviewer: I would remove that picture completely. Further it hangs around in right corner of the text. Everything you explain will be explained in the new better picture below.

Author: On the content element overview page each card has a picture and I am reusing the same picture on the top left of each page for consistency. We could maybe find a better image for this chapter.

See https://docs.typo3.org/m/typo3/tutorial-editors/13.4/en-us/ContentElements/Index.html

Author: I replaced it with one of the context menu

    :class: img-thumbnail float-end ms-1 ms-1
    :width: 250

In TYPO3, working with content happens mostly in the :guilabel:`Web > Page`
module. Chose the page you want to edit from the page tree.

..  figure:: /Images/ManualScreenshots/ContentElements/PageModule.png
    :alt: Screenshot demonstrating the page module and the page tree in the TYPO3 backend

..  contents:: Table of Content

..  _content-working-overview:

Overview of the Page module in TYPO3
====================================

> ..  figure:: ../../Images/ManualScreenshots/Page/PageModule/PageModuleColumns.png

Reviewer: This image does not exist. The complete folder PageModule does not exist

    :class: with-border

Common page icons explained:

#.  Use this icon to edit all the content elements of a column.

#.  Add a new content element to the column at this specific position.

#.  Content element icon. This can provide a visual clue to the type of content
    element. Click to access the context menu for the element (to perform
    actions like *copy*).

#.  This group of icons is enabled when hovering over the whole element.

    *  Use the pencil icon to edit the element.
    *  Enable/disable the element to show or hide it in the frontend
    *  Delete the element.

#.  If the edit icons are missing you do not have sufficient permission to
    edit the content element in question.

..  _content-working-hiding:

Hiding content elements
=======================

You can temporarily hide content that should be hidden for a while but will be
needed later. You can also hide a content element while you are still working
on it. Hidden content elements are not visible in the frontend

..  figure:: /Images/ManualScreenshots/ContentElements/HideContent.png
    :alt: A content element in the page module is hidden

>     Use the "hide" button to hide a content elements. The second content element is hidden and can be unhidden by the same button.

Reviewer: ```suggestion
    Use the "hide" button to hide a content element. The second content element is hidden and can be unhidden by the same button.
```

A content element can also be hidden in the "Access" tab of its edit form, its
context menu or the `List module <https://docs.typo3.org/permalink/t3editors:using-the-list-module-effectively>`_.

..  _content-working-moving:

Moving content elements
=======================

You can move content elements using drag and drop. You can either drop them
in one of the highlighted areas on the current page or drop them on a different
page within the page tree.

..  figure:: /Images/ManualScreenshots/ContentElements/MoveContentElement.png
    :alt: Screenshot of moving a content element via drag and drop in the TYPO3 backend

    Drop the content element on a page in the page tree to move it to another page

..  _content-working-copy-paste:

> Copy and page content elements

Reviewer: ?!?1

Copy & Paste?
Copy "page content elements"?

> Copy and page content elements

Author: ```suggestion
Copy and paste content elements
```

==============================

..  figure:: /Images/ManualScreenshots/ContentElements/CopyContent.png
    :alt: The context menu of a content menu in the TYO3 backend

>     Use the context menu to copy a content element

Reviewer: There is no TYPO3 context menu on the content element. Only the one from browser will be shown. Do you mean the context menu when clicking on the upper left icon of an content element?
```suggestion
    Use the context menu of the upper left button of the content element to start copying a content element
```

..  _content-working-life-search:

Finding content using the Live Search
=====================================

> Use the search button on the top left of the TYPO3 backend or the keyboard

Reviewer: Left?
```suggestion
Use the search button on the top right of the TYPO3 backend or the keyboard
```

> shortcut :kbd:`Ctrl + K` to open the life search.

Reviewer: ```suggestion
shortcut :kbd:`Ctrl + K` or :kbd:`CMD + K` to open the live search.
```

..  figure:: /Images/ManualScreenshots/ContentElements/LiveSearch.png
    :alt: Using the live search to search for content elements

>     Use the arrow to get more information on the search result

Reviewer: ```suggestion
    Use the arrow button to get more information on the search result
```

You can then edit the content element containing the text you were searching for.

> ..  figure:: /Images/ManualScreenshots/ContentElements/PasteContent.png

Reviewer: "paste"?

    :alt: Location of the paste button beside the "Create new content" button

    Paste the content into the desired location.

> You can also copy content elements by pressing :kbd:`Ctrl` during drag and drop.

Reviewer: Hui. We are in section "search". Seems that part should be moved to copy section

Author: Seems like I accidently moved something, will have another look at this PR, thanks for catching it

Author: Moved it back in the copy chapter

via drag an drop to the desired location or delete it, if it is not needed anymore.

..  figure:: /Images/ManualScreenshots/ContentElements/MoveUnusedContent.png
    :alt: Demonstration of moving unused content back into a normal content area

..  _content-working-troubleshooting-recycler:

Restore deleted content elements using the Recycler
---------------------------------------------------

If you accidentally delete a content element or even a complete page you can
restore your data using the module :guilabel:`Web > Recycler` if you have
sufficient permissions and it is installed.

..  figure:: /Images/ManualScreenshots/ContentElements/RestoreContent.png
    :alt: Screenshot of the Recycler module in the TYPO3 backend, demonstrating how to restore a deleted content element

See also :ref:`Recycler manual, For editors <typo3/cms-recycler:for-editors>`.

..  _content-working-troubleshooting-history:

Undo changes using the Page History
-----------------------------------

You can use the page history to undo any kind of content changes. If you have
no access to the recycler you can also use it to restore deleted content.

> Use the content menu on the page tree and choose "History/Undo".

Reviewer: ```suggestion
Use the context menu on the page tree and choose "History/Undo".
```

..  figure:: /Images/ManualScreenshots/ContentElements/RestoreContent.png
    :alt: Screenshot of the page history in the TYPO3 backend

    Use the "Undo" button to undo your changes

In this module you can even see a detailed view on how text or other fields
where changed and who made which changes when.

..  figure:: /Images/ManualScreenshots/ContentElements/ContentElementHistoryDifferences.png
    :alt: Screenshot of the page history in the TYPO3 backend

    Compare the differences. Use the undo button to undo changes.

..  _content-working-began-editing:

The BE-User 'xxx' began to edit this record x min ago
-----------------------------------------------------

It is not possible in TYPO3 for two editors to work on the same content element
at the same time. The last editor who saves the content element will override
all other changes. You should therefore avoid working on content elements that

> another editor is currently working on. These are marked with a warning:

Reviewer: Not a must have: This message will disappear after 2 hours of other users inactivity.
No need to overcomplicate things ;-)

Author: I think that will over complicate things.
